### PR TITLE
Make Servo's rounding of image-orientation values agree with Gecko's, and pass orientations directly as an enum instead of as angles.

### PR DESCRIPTION
**Pull request by jyc**
Has been reviewed: https://reviewboard.mozilla.org/r/155336/

- [ ] `./mach build -d` does not report any errors
- [ ] `./mach test-tidy` does not report any errors
- [ ] These changes fix #__ (github issue number if applicable).

- [X] There are tests for these changes OR
- [ ] These changes do not require tests because _____

Bugzilla issue: https://bugzilla.mozilla.org/show_bug.cgi?id=1355380

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/17654)

### CODE REVIEW

**Comment by jyc** [comment]
The failure is apparently something to do with LLVM? https://travis-ci.org/servo/servo/jobs/252199331#L2455

**Comment by emilio** [comment]
Travis is broken (out of disk space it seems), but it doesn't block landing.

**Comment by nox** [comment]
"Bug 1355380 - Part 2: Make Servo's rounding of image-orientation values agree with Gecko's, and pass orientations directly as an enum instead of as angles." could you being shortened.

File: components/style/properties/longhand/inherited_box.mako.rs [hunk 1]
```diff
@@ -99,31 +98,76 @@ ${helpers.single_keyword("image-rendering",
...
+                    Orientation::Angle270 => Angle::from_degrees(270.0, false),
+                }
+            }
+        }
+
+        impl ToCss for Orientation {
+            fn to_css<W>(&self, dest: &mut W) -> fmt::Result where W: fmt::Write {
+                match *self {
+                    Orientation::Angle0 => dest.write_str("0"),
+                    Orientation::Angle90 => dest.write_str("90"),
```
Comment (nox) [inline components/style/properties/longhand/inherited_box.mako.rs]: Shouldn't that be `90deg`?

Comment (jyc) [inline components/style/properties/longhand/inherited_box.mako.rs]: Yep, thanks for the catch!

**Comment by jyc** [comment]
This was backed out, and I'd like to try it again now that the full try run has passed (https://treeherder.mozilla.org/#/jobs?repo=try&revision=fd689286571af0eb0bcd1ae3255fe4e1a7230c3f)!

**Pull request merged**